### ISSUE TITLE
How do screen readers read aria-valuetext when it automatically changes?

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/membrane-transport#265]
Were were surprised to see that VoiceOver is reading the aria-valuetext whenever it changes on its own. That will impact the design. Lets learn more about it. Do other screen readers do it? Can we supress it? Do we need to leverage it?

From #263.

**Comment by jessegreenberg** [comment]
@terracoda mentioned that if the aria-valuetext cannot be controlled the way we need, we may stop using a slider.

Maybe we would use a FineCoarseSpinner that does not use AccessibleNumberSpinner.

**Comment by terracoda** [comment]
How screen readers handle changing aria-value without user interaction is an interesting question.

This is the first time we are designing description for a slider that is attached to a value that dynamically changes on its own. We need to know if we can silence aria-valuetext at convenient times, like maybe after we hear the focus event where having access to the initial value is really useful.

Can we dynamically silence aria-valuetext when we want, in order to roll our own responses that contain more relevant information about what is happening - e.g., crossing events and amount changes?

**Comment by jessegreenberg** [comment]
OK, @samreid and I tested with VoiceOver and NVDA with the following snippet. Indeed, the screen reader reads the `aria-valuetext` whenever it changes without interaction.

```html
<input id="slider" type="range" min="0" max="100" value="50" aria-valuetext="Medium">
<script>
  const s = slider;

  window.setInterval( () => {
    const value = Math.floor( Math.random() * 101 );
    const ariaValueText = value < 33 ? 'Low' : value < 66 ? 'Medium' : 'High';
    s.value = value;
    s.setAttribute( 'aria-valuetext', ariaValueText );
  }, 1500 );
</script>
```

> Can we dynamically silence aria-valuetext when we want, in order to roll our own responses that contain more relevant information about what is happening - e.g., crossing events and amount changes?

I did another test where I removed the aria-valuetext. I confirmed that NVDA does NOT read the new value. We can retest that with VoiceOver. But this approach seems tricky and there is no native way to accomplish it. How would we know when to re-apply the aria-valuetext? On the focus event? Then the `aria-valuetext` would never be read if the slider is found with the virtual cursor.

For this case, I think it would easier to not use aria-valuetext at all and use custom responses with aria-live instead.

Lets discuss again and decide a path forward together.

**Comment by jessegreenberg** [comment]
Nothing more to do here. This issue resulted in #269. Closing.
